### PR TITLE
Auth home page explore

### PR DESCRIPTION
**Pull request by sanu29**
Auth Added
Home Page Added
Explore Page added

### CODE REVIEW

File: src/components/Explore.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Box, Button, Flex, Text } from '@chakra-ui/react'
+import React, { useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { getAllVedios } from '../slice/AllVedioSlice'
```
Comment (sanjitsarkar) [inline src/components/Explore.jsx:4]: Correct the typos for getAllVedios to getAllVideos

File: src/components/Header.jsx [hunk 2]
```diff
@@ -2,10 +2,33 @@ import React from 'react'
...
+    
+   )}
+return(
+     <Link to={"/login"}>
+     <Button bgColor='teal.900' color={'white'} border={'0.5px'}
+       _hover={{color:'teal.900',bgColor:"teal.300"}}>Login</Button>
+     </Link>)
+
+}
+console.log(authState)
```
Comment (sanjitsarkar) [inline src/components/Header.jsx:31]: Remove console logging

File: src/components/Signup.jsx [hunk 3]
```diff
@@ -25,6 +25,7 @@ export  const Signup =() =>{
     dispatch(SignupThunk(userDetails))
     .unwrap()
     .then((data)=>{
+      console.log("hie")
```
Comment (sanjitsarkar) [inline src/components/Signup.jsx:28]: Remove console logging

**Review by sanjitsarkar — COMMENTED** [review]
Code is moduler and well written.
Looks good to merge.

**Review by aiman-ansari — APPROVED** [review]
Good folder structure , variable naming is also meaningful.
Remove console logs and extra spaces.
Use prettier for formatting code.

**Pull request merged**